doc/console.rst

```diff
@@ -203,7 +203,8 @@ Action parameters can be edited prior to launching it:
 
 	launcher> setname Test action that pings google.com
 
-* **settarget** sets the target of the action. The target is evaluated
+* **settarget** sets the target of the action. Targets can either be a targeting
+  string, or a macro if defined in migrc. The target is evaluated
```

Comment:
Ha, it's already here for the console :) Maybe add a small example of macro here as well?